source/api_documentation/network_integration/advertiser_campaigns/_get_advertiser_campaign_unarchive.rst

```diff
@@ -0,0 +1,15 @@
+.. container:: endpoint-long-description
```

Comment:
@spencerpbartz This comment is probably not for you, but perhaps you can pass it on to the doc owner: it seems weird to support unarchiving on a GET command. GETs typically do not alter data on the remote side.